src/GRA.Web/Views/Home/Dashboard.cshtml

```diff
@@ -285,7 +285,8 @@
                         hitType: 'event',
                         eventCategory: '@Model.SitePath',
                         eventAction: 'join',
-                        eventLabel: '@Model.ProgramName'
+                        eventLabel: '@Model.ProgramName',
+                        dimension1: '@Model.FirstTime'
```

Comment:
If the first time user tracking isn't enabled we shouldn't push anything into this dimension.